backend/apps/mentorship/management/commands/mentorship_update_comments.py

```diff
@@ -0,0 +1,152 @@
+
+    help = "Sync comments for issues relevant to active mentorship modules and process interests"
+
+    def handle(self, *args, **options) -> None:
+        """Handle the command execution."""
+        self.process_mentorship_modules()
+
+    def process_mentorship_modules(self) -> None:
+        """Process all active mentorship modules."""
+        published_modules = Module.published_modules.all()
+
+        if not published_modules.exists():
+            self.stdout.write(
+                self.style.WARNING("No published mentorship modules found. Exiting.")
+            )
+            return
+
+        self.stdout.write(self.style.SUCCESS("Starting mentorship issue processing job..."))
+
+        for module in published_modules:
+            self.stdout.write(f"\nProcessing module: {module.name}...")
+            self.process_module(module)
+
+        self.stdout.write(self.style.SUCCESS("Processed successfully!"))
+
+    def process_module(self, module: Module) -> None:
+        """Process a single mentorship module.
+
+        Args:
+            module (Module): The module instance to process.
+
+        """
+        gh = get_github_client()
+
+        module_repos = (
+            module.project.repositories.filter(id__isnull=False)
+            .values_list("id", flat=True)
+            .distinct()
+        )
+
+        if not module_repos.exists():
+            self.stdout.write(
+                self.style.WARNING(f"Skipping. Module '{module.name}' has no repositories.")
+            )
+            return
+
+        relevant_issues = Issue.objects.filter(
+            repository_id__in=module_repos, state=Issue.State.OPEN
+        ).distinct()
+
+        self.stdout.write(f"Found {relevant_issues.count()} open issues across repositories")
+
+        for issue in relevant_issues:
+            self.stdout.write(
+                f"Syncing comments for issue #{issue.number} '{truncate(issue.title, 20)}'"
+            )
+            sync_issue_comments(gh, issue)
+            self.process_issue_interests(issue, module)
+
+    def process_issue_interests(self, issue: Issue, module: Module) -> None:
+        """Process interests for a single issue.
+
+        Args:
+            issue (Issue): The issue instance to process.
+            module (Module): The module instance.
+
+        """
+        existing_interests = IssueUserInterest.objects.filter(module=module, issue=issue)
+        existing_user_ids = set(existing_interests.values_list("user_id", flat=True))
+
+        all_comments = issue.comments.select_related("author").order_by(
+            "author_id", "-nest_created_at"
+        )
+
+        interests_to_create = []
+        interests_to_remove = []
+        new_user_logins = []
+        removed_user_logins = []
+
+        user_interest_status: dict[int, dict[str, Any]] = {}
+
+        for comment in all_comments:
+            if not comment.author:
```

Comment:
What are these comments without an author? Why you didn't filter them out at L94?